WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -335,14 +342,38 @@ open class PeopleViewController: UITableViewController, NSFetchedResultsControll
             return
         }
 
-        noResultsView.titleText = NSLocalizedString("No \(filter.title) Yet",
-            comment: "Empty state message (People Management). Please, do not translate the \\(filter.title) part!")
+        noResultsView.titleText = noResultsTitle()
 
         if noResultsView.superview == nil {
             tableView.addSubview(withFadeAnimation: noResultsView)
         }
     }
 
+    private func noResultsTitle() -> String {
+        let noPeopleFormat = NSLocalizedString("No %@ Yet",
+            comment: "Empty state message (People Management). %@ can be Users or Followers")
+
+        let noPeople = String(format: noPeopleFormat, filter.title)
+
+        let noNetwork = ReachabilityUtils.noConnectionMessage()
+
+        return ReachabilityUtils.isInternetReachable() ? noPeople : noNetwork
+    }
+
+    private func handleLoadError(_ forError: Error) {
+        let _ = DispatchDelayedAction(delay: .milliseconds(500)) { [weak self] in
```

Comment:
Presenting the alert prevents the `refreshControl` from disappearing. That's why it's delayed. I saw this is happening in other lists, and to be honest I can't think of a better, simpler way to avoid that problem than delaying, that's why I followed that approach.

Do you have any other suggestions? I'm all 👂! 😛

Reply:
Nope I don't :( I tried this:
`DispatchQueue.main.async {`
but it still got stuck so I think we are good to go on this one.